lib/auto.js

```diff
@@ -2,7 +2,7 @@
 
 import arrayEach from 'lodash/_arrayEach';
 import forOwn from 'lodash/forOwn';
-import indexOf from 'lodash/_baseIndexOf';
+import indexOf from 'lodash/indexOf';
```

Comment:
We weren't importing indexof to avoid including the lodash internal
toInteger implementation
On May 4, 2016 3:53 PM, "Paul Anderson" <notifications@github.com> wrote:

> In lib/auto.js
> <https://github.com/caolan/async/pull/1140#discussion_r62103119>:
>
> > @@ -2,7 +2,7 @@
> >
> >  import arrayEach from 'lodash/_arrayEach';
> >  import forOwn from 'lodash/forOwn';
> > -import indexOf from 'lodash/_baseIndexOf';
> > +import indexOf from 'lodash/indexOf';
>
> @aearly <https://github.com/aearly> I wrote my checkForDeadlocks assuming
> I had the lodash/JavaScript standard semantics. Got test failures because I
> hadn't included start index.
>
> Thought I was better off importing the public API method, in case someone
> else were to come and repeat my mistake at another time
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/caolan/async/pull/1140/files/81e888fd40e99c0de0098a7554d6cdafc540c021#r62103119>
>